src/builders/esbuild/parse-handlers-to-entry-points.js

```diff
@@ -2,7 +2,7 @@ const parseHandlersToEntrypoints = (handlers, build) => handlers.reduce(
   (previous, handler) => {
     let outputKey = build.individually
       ? `${handler.lambda_name}/${handler.output}`
-      : `lambdas/${handler.output}`
```

Comment:
Isso não vai dar problema quando ele for procurar o entrypoint do serverless?
Confesso que não entendi tb o pq no lambdas/ aqui kkk

Reply:
eu ja adiciono o path 'lambdas' lá no inicio do processo dentro do [output](https://github.com/stone-ton/action-lambda-build/pull/14/files#diff-bfe9874d239014961b1ae4e89875a6155667db834a410aaaa2ebe3cf89820556R44)